### PR TITLE
contrib/snapshot: new package (46.3)

### PR DESCRIPTION
**Pull request by triallax**

### CODE REVIEW

File: contrib/snapshot/template.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "gst-plugins-bad-devel",
+    "gst-plugins-base-devel",
+    "libadwaita-devel",
+]
+depends = [
+    "gstreamer-libcamera",
+    "gstreamer-pipewire",
+    "libcamera",
+    "pipewire",
+    "wireplumber",
```
Comment (triallax) [inline contrib/snapshot/template.py:25]: could remove `libcamera` and `pipewire` here because they get pulled in by the other stuff

Comment (nekopsykose) [reply contrib/snapshot/template.py:25]: yeah it's fine if they crom from gst

i think you don't need wireplumber either

Comment (triallax) [reply contrib/snapshot/template.py:25]: i added wireplumber because it was mentioned as a dep but sure

Comment (nekopsykose) [reply contrib/snapshot/template.py:25]: it's a dep of anything that plays audio too, yet it's not added there either

same goes for like.. a font, an icon theme, ... base deps are not useful to add

**Pull request merged**